### PR TITLE
#15 - docker improvement, default rules set improvement

### PR DESCRIPTION
**Pull request by mtracz**
Issue #15

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -0,0 +1 @@
+CURRENT_UID=1000
```
Comment (krzysztofrewak) [inline .env.example:1]: Please add a blank line at the end of the file.

File: readme.md [hunk 2]
```diff
@@ -69,3 +69,29 @@ Then run:
...
+```
+### shell
+```shell
+docker-compose exec php ash
+```
+
+### scripts
+```shell
+docker-compose exec php composer ecs
+docker-compose exec php composer ecs-fix
```
Comment (krzysztofrewak) [inline readme.md:95]: I renamed it to `ecsf` like in other projects.

**Review by krzysztofrewak — CHANGES_REQUESTED** [review]
Please change these two commented things and resolve conflict created by my #17 PR.

File: .env.example (hunk 1 above)
Comment (mtracz) [reply .env.example]: Added.

File: readme.md (hunk 2 above)
Comment (mtracz) [reply readme.md]: Renamed

**Review by krzysztofrewak — APPROVED** [review]

**Pull request merged**